starter-code/js/index.js

```diff
@@ -1,11 +1,58 @@
-var $cart = document.querySelector('#cart tbody');
-var $calc = document.getElementById('calc');
+let $calc = document.getElementById('calc');
+let $cart = document.querySelector('#cart tbody');
 
 function updateSubtot($product) {
-  // Iteration 1.1
+  let unitPrice = parseInt($product.querySelector(".pu span").innerHTML);
+  let quantity = parseInt($product.querySelector(".qty input").value);
+  let subTotal = unitPrice * quantity;
+  return subTotal;
 }
 
 function calcAll() {
-  // Iteration 1.2
+  let productList = $cart.querySelectorAll(".product");
+  let totalPrice = 0;
+  for (i = 0; i< productList.length; i++){
+    let subTotal = productList[i].querySelector(".subtot span");
+    subTotal.innerHTML = updateSubtot(productList[i]);
+    totalPrice = totalPrice + updateSubtot(productList[i])
+  }
+  document.querySelector("#total span").innerHTML = totalPrice;
+}
+
+let calcButton = document.querySelector("#calc.btn.btn-success")
+calcButton.addEventListener("click", calcAll)
+
+let deleteButtons = $cart.querySelectorAll('.rm .btn-delete');
+for (let button of deleteButtons) {
+  button.addEventListener("click", deleteProduct);
+}
+
+function deleteProduct (event){
```

Comment:
cq--
lack of the consistency in code (space between the name of the function and () )
a proper format:
function test() {
}